src/pages/comparison/components/remove-pet-button/remove-pet-button.component.tsx

```diff
@@ -3,7 +3,7 @@ import { useDispatch } from 'react-redux';
 import { Link } from 'react-router-dom';
 import { Profile } from '../profile/profile.component';
 import { RemoveButtonProps } from './remove-pet-button.interface';
-import { removeComparisonPet } from '../../../../store/comparison/actions/comparison-handler.action'
+import { removeComparisonPet } from '../../../../store/comparison/actions/comparison-handler.action';
```

Comment:
FYI: you may import it like
'shared/store/comparison/actions/comparison-handler.action'

Reply:
right
forgot about it